api/api_types.go

```diff
@@ -242,3 +242,12 @@ func CensusTypeToOrigin(ctype CensusTypeDescription) (models.CensusOrigin, []byt
 	}
 	return origin, root, nil
 }
+
+type TokenTransfer struct {
```

Comment:
This type is declared twice. I think there is no need to declare it here (in the API package).